src/pages/course.js

```diff
@@ -190,7 +190,9 @@ const mapStateToProps = (state, ownProps) => {
   const course = state.courses.courses[ownProps.courseId]
   return {
     course,
-    queues: mapObjectToArray(state.queues.queues),
+    queues: mapObjectToArray(state.queues.queues).sort((a, b) => {
```

Comment:
just realized, I'm actually wondering if I should leave this in as these queues will be passed through the queueSorter in the QueueCardList component anyways🤔

Reply:
ended up removing this